adafruit_dht.py

```
@@ -30,8 +30,11 @@
 
 import array
 import time
-import pulseio
-
+try:
+    import pulseio
+except ImportError:
+    print("adafruit_dhtlib requires the pulseio library, but it failed to load."+
+        "  Note that CircuitPython does not support pulseio on all boards.")
```

This will hide the error. How about printing and then reraise the error?

```python
except ImportError as e:
    print("adafruit_dhtlib requires the pulseio library, but it failed to load."+
        "  Note that CircuitPython does not support pulseio on all boards.")
    raise e
```

---

Sound good.  Give me a few minutes.